src/core/sys/posix/iconv.d

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+
+    D binding for the POSIX iconv library.
+
+    See_Also:
+       http://pubs.opengroup.org/onlinepubs/009695399/functions/iconv_open.html
+
+*******************************************************************************/
+
+module core.sys.posix.iconv;
+
+version (Posix):
+extern (C):
```

Comment:
Yep, your right everything that touches mutable global state (such as `errno`) is definitely not pure.